buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -22,10 +22,10 @@ object Versions {
     internal const val ANDROIDX_TEST = "1.4.0"
     internal const val ANDROIDX_VIEW_PAGER_2 = "1.0.0"
     internal const val COIL = "1.4.0"
-    internal const val COMPOSE_ACCOMPANIST = "0.20.3"
+    internal const val COMPOSE_ACCOMPANIST = "0.23.0"
     internal const val COMPOSE_ACTIVITY = "1.4.0"
-    internal const val COMPOSE_VIEW_MODEL = "2.4.0"
-    public const val COMPOSE = "1.0.5"
+    internal const val COMPOSE_VIEW_MODEL = "2.4.1"
+    public const val COMPOSE = "1.1.0"
```

Comment:
What do you think about updating the Changlog about this? Perhaps clients may need to update the compose to `1.1.0` following our SDK updates.

Reply:
Yeah I haven't updated the hcangelog yesterday, wil do today!